docker/build-tools/Dockerfile

```diff
@@ -379,13 +390,18 @@ COPY --from=python_context /usr/local/bin /usr/local/bin
 COPY --from=python_context /usr/local/lib /usr/local/lib
 COPY --from=python_context /usr/local/google-cloud-sdk /usr/local/google-cloud-sdk
 
+# su-exec is used in place of complex sudo setup operations
+RUN chmod u+sx /usr/bin/su-exec
```

Comment:
Shouldn't this be done above, after line 163?

Reply:
no - it doesn't work unless it is in this final section.  it appears suid bits are not copied across COPY sections.